CONTRIBUTING.md

```diff
@@ -289,6 +289,8 @@ In all cases, banner text should be no more than **4** characters in length.
 
 This style strives to draw representative highway shields wherever they are tagged on road route relations consistently with international norms. This style operates on the expectation that the `network` value on a route relation corresponds to the shield design that will be drawn, and the `ref` value will contain the text which is drawn on the shield. In order to give appropriate mapper feedback, this style will add support for special cases only when the complexity of the route network and shield styling cannot be adequately expressed via `network` and `ref` alone. These special cases should be exceptionally rare and documented in the list below. PRs to add special case code should also add an entry below justifying its inclusion. For all other `network` and `ref` combinations, the style will draw a "generic" shield displaying the `ref` value.
 
+- Shields of individual routes, as designated by `ref`, having unique graphics within a shared `network`
```

Comment:
In #451, we convinced New Zealand mappers to adopt distinct `network` subvalues  for each of their touring routes, on the basis that the shields didn’t share much in common. It’s splitting hairs a bit, but I think what makes the Great Lakes Circle Tour system different is that each route retains a unifying element. Another style could reasonably represent all the routes using an identical shield featuring only the rubber band, as with the unified Oklahoma Turnpike shield we recently implemented in #733. Consider clarifying this remark to point out that each route has a unique visual element inside the shield but isn’t mostly unrelated.